source/js/components/fellow-list/fellow-list.scss

```diff
@@ -1,5 +1,7 @@
 .fellowships-directory-filter {
-  background: $subtle-gray;
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
Can this line and the previous be applied as bootstrap classes on the element? (`.d-flex` and `.flex-wrap`? (https://getbootstrap.com/docs/4.1/utilities/flex/)

Reply:
absolutely

Reply:
This is code from a different PR for the styling, but we'll make sure to take that on board there.